NEWS.md

```diff
@@ -8,6 +8,10 @@
 * `seq_linter()` additionally lints on `1:n()` (from dplyr) 
   and `1:.N` (from data.table) (#1396, @IndrajeetPatil).
 
+* `redundant_ifelse_linter()` produces correct message in the cases 
```

Comment:
much better! suggested tweak:

> correctly suggests negation when the `yes` condition is 0

(0 not in code so that it implies both the numeric and integer cases; and mentioning 1 is redundant since the lint only throws if 0/1 are paired)